src/xdocs/google_style.xml

```diff
@@ -909,36 +914,43 @@
                             </td>
                             <td>
                                 <a
-                                    href="styleguides/google-java-style-20170228.html#s4.6.2-horizontal-whitespace">
+                                    href="styleguides/google-java-style-20180523/google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace">
                                     4.6.2 Horizontal whitespace</a>
                             </td>
                             <td>
-                                <img src="images/ok_green.png" alt="" />
+                                <img src="images/ok_blue.png" alt="" />
                                 <a href="config_whitespace.html#WhitespaceAround">
                                     WhitespaceAround</a>
                                 <br />
                                 <br />
-                                <img src="images/ok_green.png" alt="" />
+                                <img src="images/ok_blue.png" alt="" />
                                 <a
                                     href="config_whitespace.html#GenericWhitespace">
                                     GenericWhitespace</a>
                                 <br />
                                 <br />
-                                <img src="images/ok_green.png" alt="" />
+                                <img src="images/ok_blue.png" alt="" />
                                 <a
                                     href="config_whitespace.html#MethodParamPad">MethodParamPad</a>
                                 <br />
                                 <br />
-                                <img src="images/ok_green.png" alt="" />
+                                <img src="images/ok_blue.png" alt="" />
```

Comment:
lets keep all non-NoWhitespaceBefore as green because "Existing Check covers all requirements from Google.".

Reply:
Well, I interpreted it as "the combination of these checks covers all requirements.
Because actually none of the single checks "covers all requirements".
That's why I switched them all to blue and maybe there should actually be only one icon per styleguide rule, not one icon per checkstyle rule.
For now I switched the others back to green as requested.